docs/source/building-supergraphs/subgraphs-apollo-server.mdx

```diff
@@ -137,24 +164,27 @@ const resolvers = {
 
 Finally, we use the `buildSubgraphSchema` function from the `@apollo/subgraph` package to augment our schema definition with federation support. We provide the result of this function to the `ApolloServer` constructor:
 
-```js title="index.js"
+<MultiCodeBlock>
+
+```ts title="index.ts"
 const server = new ApolloServer({
   schema: buildSubgraphSchema({ typeDefs, resolvers })
 });
-
-server.listen(4001).then(({ url }) => {
-    console.log(`🚀 Server ready at ${url}`);
-});
 ```
 
+</MultiCodeBlock>
+
 The server is now ready to act as a subgraph in a federated graph!
 
 ### Combined example
 
 Here are the snippets above combined (again, note that for this sample to be complete, you must define the `fetchUserById` function for your data source):
 
-```js title="index.js"
-const { ApolloServer, gql } = require('apollo-server');
+<MultiCodeBlock>
+
+```ts title="index.ts"
+const { ApolloServer } = require('@apollo/server');
+const { gql } = require('graphql-tag');
 const { buildSubgraphSchema } = require('@apollo/subgraph');
```

Comment:
Probably should convert to `import`s if we're switching it to TS (though it's also not incorrect as-is)

Reply:
Oh great catch! I converted these in the previous snippet but forgot to update the complete example. Updated!